login-workflow/src/contexts/RegistrationContext/RegistrationDictionaries/chinese.ts

```diff
@@ -23,6 +23,8 @@ const resources: RegistrationLanguageFile = {
                 '您已成功使用<b>{{email}}</b>注册了一个新账号。\n\n您的账号已被加入“<b>{{organization}}</b>”。',
             SUCCESS_MESSAGE_ALT:
                 '您已成功使用<1>{{email}}</1>注册了一个新账号。\n\n您的账号已被加入“<3>{{organization}}</3>”。',
+            SUCCESS_MESSAGE_ALT_WITHOUT_EMAIL_PROVIDED:
+                '您的帳戶已成功創建。\n\n您的账号已被加入“<3>{{organization}}</3>”。',
```

Comment:
You have a mix of Chinese traditional and Chinese simplified.
```suggestion
                '您已成功注册了一个新账号。\n\n您的账号已被加入“<3>{{organization}}</3>”。',
```